src/styles/h5p-multi-media-choice.scss

```diff
@@ -254,7 +254,8 @@
 
   .h5p-multi-media-modal {
     position: absolute;
-    background: rgba(0,0,0,0.85);
+    background-color: rgba(0,0,0,0.85); /* Fallback color */
```

Comment:
Good to have the fallback. H5P promises to support the latest three versions of major browsers and those should not have trouble with relative hsl colors. But again: good to have it.